runtime/src/main/java/org/corfudb/protocols/logprotocol/LogEntry.java

```diff
@@ -83,7 +74,7 @@ public static ICorfuSerializable deserialize(ByteBuf b, CorfuRuntime rt) {
      *
      * @param b The remaining buffer.
      */
-    void deserializeBuffer(ByteBuf b, CorfuRuntime rt) {
+    void deserializeBuffer(ByteBuf b) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651473)